.github/workflows/codespell.yml

```
@@ -1,6 +1,9 @@
 name: Codespell
 on:
   pull_request:
+permissions:  
```

To keep space coherency

```suggestion

permissions:
```